lib/ansible/modules/cloud/amazon/iam_server_certificate_info.py

```diff
@@ -10,22 +10,28 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: iam_server_certificate_info
 short_description: Retrieve the information of a server certificate
 description:
-  - Retrieve the attributes of a server certificate.
+  - Retrieve the attributes of a server certificate
   - This module was called C(iam_server_certificate_facts) before Ansible 2.9. The usage did not change.
 version_added: "2.2"
-author: "Allen Sanabria (@linuxdynasty)"
+author: "Vijayanand Sharma (@vijayanandsharma)"
 requirements: [boto3, botocore]
 options:
   name:
     description:
-      - The name of the server certificate you are retrieving attributes for.
+      - The name of the server certificate you are retrieving attributes for. This argument is mutually exclusive with I(path)
```

Comment:
```suggestion
      - The name of the server certificate you are retrieving attributes for.
      - This argument is mutually exclusive with I(path).
```

1. Rather than creating a long line, this can be split into two lines.
2. Please use full sentences (including a full stop) for descriptions.